hnn_core/optimization/metrics.py

```diff
@@ -0,0 +1,72 @@
+"""Metrics for parameter optimization."""
+
+# Authors: Carolina Fernandez <cxf418@miami.edu>
+#          Nick Tolley <nicholas_tolley@brown.edu>
+#          Ryan Thorpe <ryan_thorpe@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+
+import numpy as np
+
+from hnn_core import simulate_dipole
+
+from scipy.signal import resample
+
+
+def _rmse_evoked(net, initial_params, set_params, predicted_params,
```

Comment:
@carolinafernandezp I commented on this in the previous iteration. There is already: https://github.com/jonescompneurolab/hnn-core/blob/c81e5b87fbcc13bd4c6e5a403e838ca63ea16760/hnn_core/dipole.py#L239

Why not reuse that? 

As a general rule, you should address all the comments and/or provide an explanation why it could not be addressed ... otherwise, the PR will remain in a stale state as reviewers will see the same issues again.

Reply:
Let me know if I'm not understanding correctly: 
- We don't pass in dpl, dpl is computed inside _rmse_evoked.
- We talked about using 'target' instead of 'exp_dpl' because the user will not always pass a dipole to the fit method (although 'target' will become optional once we introduce the rhythmic metric, so we could go ahead and call it 'exp_dpl').
- We don't use 'tstart' or 'weights' (I think the old routine used those because of the chunking approach).

Reply:
Just to recapitulate what @carolinafernandezp said in different words, `_rmse_evoked()` is an objective function that simulates `net` from within. @jasmainak maybe there's a way to clarify this by renaming this module `objective_functions.py`?

Reply:
rename for sure but let me point out the common code so you can see what I'm saying

Reply:
also there is a `resample` ... it's not the same as downsample as it also lowpass filters. I haven't closely followed. Is that what you intend to do? The filtering step could slow your computation

Reply:
In reference to `signal.resample`, we're always downsampling, either the target dpl time series or the experimental dpl time series (whichever has more data points). I think we do this also in `dipole.py` which is why I though it was appropriate. The only other function I know is `scipy.signal.decimate` which filters the signal before downsampling. Is there another that's more appropriate?